groovy/postInit/chemistry/inorganic_chemistry/elements/d_block/group7/ManganeseChain.groovy

```diff
@@ -131,6 +133,32 @@ ELECTROLYTIC_CELL.recipeBuilder()
     .EUt(Globals.voltAmps[2])
     .buildAndRegister()
 
+// MnSO4
+
+BR.recipeBuilder()
+    .inputs(ore('dustManganeseIiOxide') * 2)
```

Comment:
you haven't added a recipe to make purified MnO

Reply:
done